lib/two_factor_authentication/models/two_factor_authenticatable.rb

```diff
@@ -87,6 +103,19 @@ def generate_totp_secret
           ROTP::Base32.random_base32
         end
 
+        def create_backup_codes(options = {})
```

Comment:
Wanna document the usage of this method in the README? It's only being called in the tests in this PR.